### PR TITLE
PMM-9246 Extended MongoDB message

### PR DESCRIPTION
**Pull request by percona-csalguero**
Added `stats_collections` and `collections_limit` to the MongoDB message type for invetory

### CODE REVIEW

File: api/inventorypb/agents.proto [hunk 1]
```diff
@@ -158,6 +158,11 @@ message MongoDBExporter {
   AgentStatus status = 9;
   // Listen port for scraping metrics.
   uint32 listen_port = 10;
+  // List of colletions to get stats from. Can use *
+  repeated string stats_collections = 13;
+  // Collections limit. Only get Databases and collection stats if the total number of collections in the server
```
Comment (JiriCtvrtka) [inline api/inventorypb/agents.proto:163]: In which units Limit is?

Comment (percona-csalguero) [reply api/inventorypb/agents.proto:163]: number of collections

File: api/managementpb/json/managementpb.json [hunk 2]
```diff
@@ -2528,6 +2528,12 @@
                       "type": "string",
                       "x-order": 0
                     },
+                    "collections_limit": {
+                      "type": "integer",
+                      "format": "int32",
+                      "title": "Collections limit. Only get Databases and collection stats if the total number of collections in the server\nis less than this value. 0: no limit",
```
Comment (ShashankSinha252) [inline api/managementpb/json/managementpb.json:2534]: `description` instead of `title`, for sake of consistency?

**Review by JiriCtvrtka — APPROVED** [review]
LGTM just do changes addressed by Shashank.

File: api/inventorypb/agents.proto [hunk 3]
```diff
@@ -158,6 +158,11 @@ message MongoDBExporter {
...
+  // is less than this value. 0: no limit
+  int32 collections_limit = 14;
```
[the remark this answers is not in the document]
Comment (percona-csalguero) [reply api/inventorypb/agents.proto:165]: Because we could use -1 to disable

File: api/managementpb/json/managementpb.json (hunk 2 above)
Comment (percona-csalguero) [reply api/managementpb/json/managementpb.json:2534]: This is auto generated by swagger libs

File: api/inventorypb/agents.proto (hunk 3 above)
[the remark this answers is not in the document]
Comment (ShashankSinha252) [reply api/inventorypb/agents.proto:165]: Got it.

**Review by ShashankSinha252 — APPROVED** [review]
LGTM

**Pull request merged**